crates/bevy_audio/src/audio.rs

```diff
@@ -3,26 +3,36 @@ use bevy_asset::{Asset, Handle};
 use parking_lot::RwLock;
 use std::{collections::VecDeque, fmt};
 
-/// The external struct used to play audio
-pub struct Audio<P = AudioSource>
+/// Use this resource to play audio
+///
+/// ``` edition2021
```

Comment:
Genuine question: why do we need the `edition2021` label here?

Reply:
I guess I've been adding those out of habit. The rust docs book doesn't say anything about the default, bu I would hope that it's the same as cargos' default.

The doc tests are still green without the labels, so I removed them.

Reply:
I just found a reason for keeping the labels and was reminded of this comment. Docs.rs [renders an info icon next to inline examples](https://docs.rs/bevy_asset_loader/0.6.0/bevy_asset_loader/).
In case an example is written in an older edition that info might be helpful. For example, the Bevy examples currently on docs.rs are written with edition2018.